content/en/docs/armory-agent/agent-mtls.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "Configure Mutual TLS Authentication"
+linkTitle: "Configure mTLS"
+weight: 40
+description: >
+  Configure mTLS authentication in the Armory Agent plugin and service.
+---
+
+## {{% heading "prereq" %}}
+
+You need the following to configure mTLS:
+
+* `CA.crt`
```

Comment:
I was looking at the preview of the published version and I think we need to be more clear. "CA.crt" doesn't mean a lot outside of the team I think. It would probably be better to say "A Certificate Authority certificate in the pem format, used for validating the issuer for mTLS requests."